packages/sdk/src/swap/services/ApiService.ts

```diff
@@ -23,18 +23,31 @@ const getChains = async (network: ChainflipNetwork): Promise<ChainData[]> => [
   ethereum(network),
   polkadot(network),
   bitcoin(network),
+  arbitrum(network),
 ];
 
 const getPossibleDestinationChains = async (
   sourceChain: Chain,
   network: ChainflipNetwork,
 ): Promise<ChainData[]> => {
   if (sourceChain === Chains.Ethereum)
-    return [ethereum(network), bitcoin(network), polkadot(network)];
+    return [
+      ethereum(network),
+      bitcoin(network),
+      polkadot(network),
+      arbitrum(network),
```

Comment:
maybe include this only for `backspin` for now? otherwise partners will already get the asset/chain even if it snot supported by `partnernet`